introduction_to_amazon_algorithms/jumpStart_tabular_classification/Amazon_JumpStart_Tabular_Classification_LightGBM_CatBoost.ipynb

```diff
@@ -0,0 +1,596 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    predictor.delete_model()
</pre><p>Do we need to delete the endpoint config / s3 assets too?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>